packages/docs/cookbook/migration-vuex.md

```diff
@@ -0,0 +1,271 @@
+# Migrating from Vuex
+
+Although the structure of Vuex and Pinia stores is different, a lot of the logic can be reused. This guide serves to help you through the process and point out some common gotchas that can appear.
+
+## Preparation
+
+First, follow the [Getting Started guide](../getting-started.md) to install Pinia.
+
+## Restructuring Modules to Stores
+
+Vuex has the concept of a single store with multiple _modules_. These modules can optionally be namespaced and even nested within each other.
+
+The easiest way to transition that concept to be used with Pinia is that each module you used previously is now a _store_. Each store has an `id` which is similar to a namespace. Nested modules can also each become their own store. Stores that depend on each other will simply import the other store.
+
+How you choose to restructure your Vuex modules into Pinia stores is entirely up to you, but here is one suggestion:
+
+```sh
```

Comment:
I used this so that the comments would appear dimmer, which works in my IDE but not on the actual site. Changing this to `bash` does though, see what you think. With no language it all appears white which makes it a little harder to read IMO.